src/Manager/managerSlice.test.ts

```diff
@@ -61,6 +61,8 @@ describe('managerReducer', () => {
             addEnvironment(anEnvironment),
         ]);
 
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
@kylebonnici Please try to avoid using `@ts-ignore`. In most cases `@ts-expect-error` is better. Even better: Add an explanation to why an error here is expected. E.g. here this might have been good:

```
// @ts-expect-error -- The type RootState wrongly still contains `appReloadDialog` in shared as of v6.6.4
```

This has several advantages: 
- You do not need to also disable the `@typescript-eslint/ban-ts-comment` rule
- When the problem is fixed, the check fails, so you know you must remove the `@ts-expect-error` (Less outdated disables)
- When reading the code, it is easier to understand why the warning was disabled

Because the underlying problem was fixed in `shared`, I removed the disables now in https://github.com/NordicSemiconductor/pc-nrfconnect-toolchain-manager/pull/184/commits/bed2d67a539cc0298d86bd79a90a861d57044fe4.